doc/articles/features/orientation-sensor.md

```diff
@@ -5,34 +5,34 @@ uid: Uno.Features.OrientationSensor
 # Orientation sensor
 
 > [!TIP]
-> This article covers Uno-specific information for SimpleOrientationSensor. For a full description of the feature and instructions on using it, consult the UWP documentation: https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.simpleOrientationSensor
+> This article covers Uno-specific information for SimpleOrientationSensor. For a full description of the feature and instructions on using it, consult the UWP documentation: <https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.simpleOrientationSensor>
 
- * The `Windows.Devices.Sensors.SimpleOrientationSensor` class allows you to determine the general orientation of the device.
+* The `Windows.Devices.Sensors.SimpleOrientationSensor` class allows you to determine the general orientation of the device.
 
 ## Supported features
 
-| Feature        |  Windows  | Android |  iOS  |  Web (WASM)  | macOS | Linux (Skia)  | Win 7 (Skia) | 
+| Feature        |  Windows  | Android |  iOS  |  Web (WASM)  | macOS | Linux (Skia)  | Win 7 (Skia) |
 |---------------|-------|-------|-------|-------|-------|-------|-|
 | `GetDefault`         | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
-| `ReadingChanged` | ✔ | ✔ | ✔ | ✔ | ✖ | ✖| ✖ |
+| `OrientationChanged` | ✔ | ✔ | ✔ | ✔ | ✖ | ✖| ✖ |
 
 ## Using SimpleOrientationSensor with Uno
- 
- * The `GetDefault` method is available on all targets and will return `null` on those which do not support `SimpleOrientationSensor` or devices that do not have such a sensor.
- * Ensure to unsubscribe from the `ReadingChanged` event when you no longer need the readings, so that the sensor is no longer active to avoid unnecessary battery consumption.
+
+* The `GetDefault` method is available on all targets and will return `null` on those that do not support `SimpleOrientationSensor` or devices that do not have such a sensor.
```

Comment:
Maybe we can explicitly say that the simulator doe not support it?

```suggestion
* The `GetDefault` method is available on all targets and will return `null` on those that do not support `SimpleOrientationSensor` or devices that do not have such a sensor (e.g. the iOS Simulator).
```

Reply:
The thing is that `GetDefault()` still returns an instance, not null,  but the `OnOrientationChanged` event never triggers. Making `GetDefault()` return null on devices that do not support (including the simulators) is the right move, but it's a breaking change. 

@jeromelaban what's your suggestion?

Reply:
That's indeed interesting. We can make that change for Uno 5, we already have other related breaking changes.

Reply:
Copied!